google_photos_takeout_helper/__main__.py

```diff
@@ -425,6 +425,13 @@ def degToDmsRational(degFloat):
 
         return [(deg, 1), (deg_min, 1), (sec, 100)]
 
+    # converts a string input into a float. If it fails, it returns 0.0
+    def string_to_float(num):
+        if type(num) == str:
+            return 0.0
+        else:
+            return float(num)
+
```

Comment:
Can you move this inside `set_file_geo_data`, just above it's usage  and rename to `_str_to_float`? So it's more local

Reply:
Done.